frontend/src/components/widgets/UserAvatarLink.jsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import Avatar from "@material-ui/core/Avatar";
+import Typography from "@material-ui/core/Typography";
+import { makeStyles } from "@material-ui/styles";
+import { getUrl as getUserBoardUrl } from "./LinkToUserBoard";
+
+const useStyles = makeStyles({
+  avatarLink: {
+    textDecoration: "none",
+  },
+  container: {
+    display: "flex",
+    alignItems: "center",
+    gap: 8,
+  },
+});
+
+export function getAvatarTextInitials(email) {
```

Comment:
Great work,  just a bit concerned about the choice of names (variables  and functions)

Shouldn't this be rather getAvatarInitials?